[2024-08-29T09:23:06.079+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-08-29T09:23:06.108+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Spark.geo_map_plot manual__2024-08-29T09:21:30.526177+00:00 [queued]>
[2024-08-29T09:23:06.127+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Spark.geo_map_plot manual__2024-08-29T09:21:30.526177+00:00 [queued]>
[2024-08-29T09:23:06.128+0000] {taskinstance.py:2856} INFO - Starting attempt 1 of 1
[2024-08-29T09:23:06.158+0000] {taskinstance.py:2879} INFO - Executing <Task(PythonOperator): geo_map_plot> on 2024-08-29 09:21:30.526177+00:00
[2024-08-29T09:23:06.169+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'Spark', 'geo_map_plot', 'manual__2024-08-29T09:21:30.526177+00:00', '--job-id', '326', '--raw', '--subdir', 'DAGS_FOLDER/Seoul_spark.py', '--cfg-path', '/tmp/tmpl9e9y0m_']
[2024-08-29T09:23:06.176+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=2632) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-08-29T09:23:06.178+0000] {standard_task_runner.py:105} INFO - Job 326: Subtask geo_map_plot
[2024-08-29T09:23:06.178+0000] {standard_task_runner.py:72} INFO - Started process 2634 to run task
[2024-08-29T09:23:06.281+0000] {task_command.py:467} INFO - Running <TaskInstance: Spark.geo_map_plot manual__2024-08-29T09:21:30.526177+00:00 [running]> on host 1c1d52fdfc5f
[2024-08-29T09:23:06.467+0000] {taskinstance.py:3122} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='Spark' AIRFLOW_CTX_TASK_ID='geo_map_plot' AIRFLOW_CTX_EXECUTION_DATE='2024-08-29T09:21:30.526177+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-08-29T09:21:30.526177+00:00'
[2024-08-29T09:23:06.469+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-08-29T09:23:12.077+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/pyspark/pandas/__init__.py:50 UserWarning: 'PYARROW_IGNORE_TIMEZONE' environment variable was not set. It is required to set this environment variable to '1' in both driver and executor sides if you use pyarrow>=2.0.0. pandas-on-Spark will set it for you but it does not work if there is a Spark context already launched.
[2024-08-29T09:24:55.712+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-08-29T09:24:55.825+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-08-29T09:24:55.827+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=Spark, task_id=geo_map_plot, run_id=manual__2024-08-29T09:21:30.526177+00:00, execution_date=20240829T092130, start_date=20240829T092306, end_date=20240829T092455
[2024-08-29T09:24:55.922+0000] {local_task_job_runner.py:261} INFO - Task exited with return code 0
[2024-08-29T09:24:56.034+0000] {taskinstance.py:3891} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-08-29T09:24:56.036+0000] {local_task_job_runner.py:240} INFO - ::endgroup::
